ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -319,6 +332,39 @@ tests:
   steps:
     workflow: stackrox-automation-flavors-rosa-e2e
   timeout: 5h0m0s
+- as: ocp-next-qa-e2e-tests
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:ec
+    env:
+      TEST_SUITE: openshift-newest-qa-e2e-tests
```

Comment:
I wonder why did we have to create symlinks in `stackrox/stackrox` if we could just use this to override job name.